src/lein_monolith/task/graph.clj

```diff
@@ -9,6 +9,11 @@
 
 (def image-name "project-hierarchy.png")
 
+(defn cluster->descriptor
+  [monolith-root subdirectory]
+  (when-not (= monolith-root subdirectory)
```

Comment:
While I was writing it, I debated whether to use `not=`- seems I picked wrong, haha.

I've corrected the spacing, as well.